### DISCUSSION TITLE
Tagging of reusable workflows?

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [community/community#26798] [Actions]
<p>What does the <code>@{ref}</code> reference ?</p>
<p>What does <code>@v1</code> reference ? GitHub tag ?</p>
<p>What does <code>@master</code> reference ? Branch ?</p>
<p>What if I want to reference the reusable workflow from the exact branch where the workflow that is reusing the first one is executed ?</p>

**Comment by jenschelkopf** [comment, marked as answer]
<p>Hi <a class="mention" href="/aleks-ivanov">@aleks-ivanov</a>. The <code>{ref}</code> can be a SHA, a release tag, or a branch name. Today, there’s not a way to reference the reusable workflow by a dynamic value (such as the current ref_head). Here’s another thread about that: <a href="https://github.community/t/ref-head-in-reusable-workflows/203690" class="inline-onebox">Ref Head in Reusable Workflows</a></p>
